content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -5,44 +5,50 @@ index: 1
 
-The methods provided for specifying channel parameters, and the currently available features, are outlined below.
+The methods provided for specifying channel parameters are outlined below.
 
-h2(#supported-params). Currently supported channel params
+h2(#supported-parameters). Supported channel parameters
 
-- rewind := Used to request that an attachment start from some number of messages or point in time in the past. See "rewind":#rewind for more information
-- delta := **In an experimental state**. Used to request that data payloads should be sent as deltas to the previous payload. "Contact us":https://www.ably.io/contact for more information and supported values
+- rewind := Used to request that an attachment start from a given number of messages or point in time in the past. See "rewind":/realtime/channels/channel-parameters/rewind for more information.
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "deltas":/realtime/channels/channel-parameters/deltas for more information.
 
-h2(#using-params-with-lib-v11). Using channel params with v1.1 or earlier Ably client libraries
+h2(#using-parameters). Using channel parameters
 
-The current Ably libraries, at version 1.1, do not expose an API for expressing channel parameters. This means that it is necessary to specify parameters in a way that is opaque to the library.
-
-A set of params is expressed by including a query string, using standard URL query syntax and encoding, within the qualifier part of a channel name. The qualifier part is in square brackets at the start of the channel name.
-
-For example, to specify the parameter @foo@ with value @bar@ on channel @baz@, the qualified channel name would be @[?foo=bar]baz@. If the channel name already has a qualifier, such as @[meta]log@, then the query string follows the existing qualifier, as in @[meta?foo=bar]log@.
-
-Using this syntax with the Ably library means that channel parameters are specified for the lifetime of the @Channel@ instance; in order to reference the same channel, but with different channel parameters, it is necessary to get a new @Channel@ instance, using a qualified name that includes the new channel parameters.
+You can specify channel parameters in the @ChannelOptions@ when obtaining a @Channel@. A collection of channel parameters is expressed as a map of string key/value pairs. The @ChannelOptions@ associated with a channel may also be updated by calling "setOptions":/realtime/channels/overview#modifying-options. The parameters associated with a channel take effect when the channel is first attached; if the parameters are subsequently modified via a call to @setOptions@, then that call triggers an attach operation that applies the updated parameters, if successful.
 
 h3. Example
```

Comment:
@QuintinWillison / @paddybyers do we want to add a simple deltas example here as well?

Reply:
I don't mind either way - I don't think it's necessary. There is a link just above to the deltas page where there are plenty of examples.